docs/source/reference/environment.rst

```diff
@@ -46,10 +46,13 @@ Here are the environment variables CuPy uses.
 For install
 -----------
 
-These environment variables are only used during installation.
-
-+---------------+---------------------------------------------------------------------+
-| ``CUDA_PATH`` | See the description above.                                          |
-+---------------+---------------------------------------------------------------------+
-| ``NVCC``      | Define the compiler to use when compiling CUDA files.               |
-+---------------+---------------------------------------------------------------------+
+These environment variables are used during installation (building CuPy from source).
+
++-------------------+---------------------------------------------------------------------+
+| ``CUDA_PATH``     | See the description above.                                          |
++-------------------+---------------------------------------------------------------------+
+| ``CUTENSOR_PATH`` | Path to the cuTENSOR v0.2+ root containing ``lib`` and ``include``  |
```

Comment:
Is the minimum version relevant to paths to `lib` and `include`?

Reply:
No, it's just to indicate the minimum required version in https://github.com/cupy/cupy/pull/2341.

Reply:
How about adding the info to `docs/source/install.rst#optional-libraries`?